cmd/m3/admin-add.go

```diff
@@ -0,0 +1,82 @@
+// This file is part of MinIO Kubernetes Cloud
+// Copyright (c) 2019 MinIO, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/minio/cli"
+	"github.com/minio/m3/cluster"
+)
+
+// Adds a user to the tenant's database
+var adminAddCmd = cli.Command{
+	Name:   "add",
+	Usage:  "Adds an admin to the defined tenant",
+	Action: adminAdd,
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "name",
+			Value: "",
+			Usage: "admin name",
+		},
+		cli.StringFlag{
+			Name:  "email",
+			Value: "",
+			Usage: "user email",
+		},
+	},
+}
+
+// adminAdd is a command to add a cluster admin.
+// sample usage:
+//     m3 admin add "User Name" user@acme.com
```

Comment:
can you please add this new feature to the readme?

Reply:
done